aggregate/central-site/pom.xml

```diff
@@ -8,7 +8,7 @@
     <version>4.3.0-SNAPSHOT</version>
   </parent>
   <artifactId>org.jboss.tools.site.central</artifactId>
-  <name>JBoss Tools - Central</name>
+  <name>JBoss Central</name>
```

Comment:
This is only accidental is it not ? i.e. IS central is different between community and product versions; same would/should happen for jbosstools/devstudio.